File: Backend/RoomMe/RoomMe.API/Models/HouseworkFullGetModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace RoomMe.API.Models
{
    public class HouseworkFullGetModel
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public FlatShortModel Flat { get; set; }
        public UserNicknameModel Author { get; set; }
        public string Description { get; set; }
        public List<UserNicknameModel> Users { get; set; }
        public List<ScheduleDateModel> Schedules { get; set; }
    }
}

File: Android-app/domain/src/main/java/uj/roomme/domain/housework/HouseworkFullGetModel.kt
package uj.roomme.domain.housework

import uj.roomme.domain.flat.FlatShortModel
import uj.roomme.domain.schedule.ScheduleDateModel
import uj.roomme.domain.user.UserNicknameModel

data class HouseworkFullGetModel(
    val id: Int,
    val name: String,
    val flat: FlatShortModel,
    val author: UserNicknameModel,
    val description: String,
    val users: List<UserNicknameModel>,
    val schedules: List<ScheduleDateModel>
)
